### DISCUSSION TITLE
Audio Inputs

### DISCUSSION THREAD

**Discussion by geoff-maddock** [noisedeck/noisedeck-docs#9] [Questions and Issues] [answered]
I drilled down into the settings that allow you to use audio as an input to a parameter.  
Stoked to see that, and something I'll def use!  

I was wondering if we could add a level meter somewhere in the UI (left side?) that could be turned on/off, so we can see what the app is using as a reference value?  Might be extraneous, but I could see it being useful.

**Comment by jesshewitt** [comment, marked as answer]
That's reasonable! I'll add it to the feature requests. It's very early days for the audio support and we would welcome any ideas or feedback you may have.
